pkg/trace/obfuscate/sql_test.go

```diff
@@ -14,13 +14,20 @@ import (
 
 	"github.com/DataDog/datadog-agent/pkg/trace/pb"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 type sqlTestCase struct {
 	query    string
 	expected string
 }
 
+type sqlTestCaseNormalize struct {
```

Comment:
These two structs are too similar, can't we just modify the other one by adding `excpectedNew string` field? Should be a piece of cake.